modular_skyrat/modules/aesthetics/morgue/code/morgue.dm

```diff
@@ -1,2 +1,6 @@
 /obj/structure/bodycontainer
 	icon = 'modular_skyrat/modules/aesthetics/morgue/icons/morgue.dmi'
+
+/obj/item/paper/guides/jobs/medical/morgue
+	name = "morgue memo"
+	default_raw_text = "<font size='2'>Since this station's medbay never seems to fail to be staffed by the mindless monkeys meant for genetics experiments, I'm leaving a reminder here for anyone handling the pile of cadavers the quacks are sure to leave.</font><BR><BR><font size='4'><font color=magenta>Purple lights mean there's a plain ol' dead body inside.</font><BR><BR><font color=orange>Yellow lights mean there's non-body objects inside.</font><BR><font size='2'>Probably stuff pried off a corpse someone grabbed, or if you're lucky it's stashed booze.</font><BR><BR><font color=red>Red lights mean the morgue system detects the body may be able to be brought back to life.</font></font><BR><font size='2'>I don't know how that works, but keep it away from the kitchen and go yell at the geneticists.</font><BR><BR>- CentCom Medical Inspector"
```

Comment:
I think I'd rather have green as the color for a body that can be revived, I think it'd make more sense than red. Other than that, I'm happy to see this.

Reply:
My thought that a flashing red light as it audibly beeps conveys 'you did it wrong, there's someone revivable in here'

Green seems like it means you did everything fine.

Reply:
On second look, TG uses green for that too, which I was trying to make more consistent, so yeah I'll swap red to green.